src/pastel/library/Pastel.re

```diff
@@ -5,47 +5,47 @@
  * LICENSE file in the root directory of this source tree.
  */;
 
-open CreateChalker;
+open MakePastel;
 
-type chalker = string => string;
+type pastel = string => string;
```

Comment:
I don't think this type should be named after the framework, but rather what it's doing. "Chalker" wasn't a good name either

Perhaps `decorator` or `stringDecorator`?

Reply:
I like decorator